companies/moebel-kraft.json

```diff
@@ -0,0 +1,20 @@
+{
+    "slug": "moebel-kraft",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "commerce"
+    ],
+    "name": "Möbel-Kraft Online GmbH & Co. KG",
+    "address": "Am Rondell 1\n12529 Schönefeld ",
+    "phone": "49 (0)4551 / 50 -1",
+    "email": "onlineservice@moebel-kraft.de",
+    "web": "https://www.moebel-kraft.de",
```

Comment:
```suggestion
    "fax": "+49 08003522222",
    "web": "https://www.moebel-kraft.de",
```
added generic fax
I'm not quite sure, if this is the correct international format for: +49 (0) 0800 - 35 22 222
source: [https://www.moebel-kraft.de/impressum](https://www.moebel-kraft.de/impressum)

Reply:
It should be `+49 800 3522222` (I'll add that above).

We use https://libphonenumber.appspot.com/ ("International format" under "Formatting Results" on the results page) as our canonical source for phone number formatting (but additional spaces can be added in the last block for easier readability).

Reply:
I have never seen 00800 as prefix before, only 0800 for toll-free numbers (or any area code/prefix with double 0)...

Reply:
I think that's a typo on the site. Pretty sure that isn't valid otherwise.

Reply:
I just found out that 00800 actually exists 👀 , see [00800: Free­phone inter­national (teltarif.de)](https://www.teltarif.de/i/sonderrufnummern-0800.html)

Reply:
Interesting, didn't know that either.

Reply:
Anyhow, no idea how you would call 00800 from abroad. +49 0800 or +49 800 ...